### ISSUE TITLE
[Content styleguide] Labels or instructions aren't provided when content requires user input. (05.11.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#29359]
#### VFS team name
Education Application (BAH)

#### VFS product name
Comparison Tool Redesign

#### Point of Contact/Reviewers
Trevor Pierce (Accessibility)
Peggy Gannon (Content)

---

### Platform Issue
Labels or instructions aren't provided when content requires user input.

### Issue Details
Search by location tab: The city, state, postal code input must have a label

Seach by name tab: School, employer, or training provider doesn't have a label

### Link, screenshot or steps to recreate
![Screen Shot 2021-08-26 at 5 22 19 PM](https://user-images.githubusercontent.com/934879/131043820-1b586cad-1a10-4084-b91b-797fea54f242.png)

---
![Screen Shot 2021-08-26 at 5 22 24 PM](https://user-images.githubusercontent.com/934879/131043831-7baca321-e5c6-4de3-889a-9bc36c5b75ca.png)

### VA.gov Experience Standard
[Cateogy Number 05, Issue Number 11](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.3.2_A

---

### Platform Recommendation
Add a visual label (ideal) or an aria-label to the input

### Acceptance Criteria:
1. For search by name:
    a. There is no helper text within the search field input.
    b. The search field input has the following label: `School, employer, or training provider`.
    c. The input is a required field.
    d. The following validation message is displayed when trying to execute a search without entering information: `Please fill in a school, employer, or training provider.`

2. For search by location:
    a. There is no helper text within the search field input.
    b. The search field input has the following label: `City, state, or postal code`.
    c. The input is a required field.
    d. The following validation message is displayed when trying to execute a search without entering information: `Please fill in a city, state, or postal code.`

### Supporting Artifacts:
1. [Desktop prototype](https://bahdigital.invisionapp.com/share/QCIAGXOTD3P#/296063000_Searchbyname_Desktop_5)
2. [Mobile prototype](https://bahdigital.invisionapp.com/share/QDIAGYESX4T#/296060491_CT_Searchbyname_mobile_1)

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by amyknox** [comment]
@dneel-bah 

[Desktop prototype here](https://bahdigital.invisionapp.com/share/QCIAGXOTD3P#/296063000_Searchbyname_Desktop_5)

Fix: Remove text from inside the text field and add a label above the text field.  Per Monday's discussion, we added (*Required) to these fields so an error message will display if a user attempts to search with an empty field.

ERROR MESSAGES
Name search error message: Please fill in a school, employer, or VET TEC provider.
Location search error: Please fill in a city, state, or postal code.

**Comment by dmccurdybah** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/18522

**Comment by mattpointzxer0** [comment]
Lookin' good!
